pkg/uki/install.go

```diff
@@ -35,8 +36,13 @@ func (i *InstallAction) Run() (err error) {
 
 	// immucore mounts CDROM under this path
 	_, err = i.cfg.Fs.Stat(constants.UkiCdromSource)
-	if err != nil {
-		i.cfg.Logger.Errorf("Error checking if %s exists: %s", constants.UkiCdromSource, err)
+	if errors.Is(err, os.ErrNotExist) {
+		// The directory is under /run, so we can create it if not present
+		if err := fsutils.MkdirAll(i.cfg.Fs, constants.UkiCdromSource, os.ModeDir|os.ModePerm); err != nil {
```

Comment:
no, this is wrong, if that path doesnt exists it means that immucore didnt mount the cdrom under there, so the next step wont work. We NEED that path mounted to be able to mount the following image, creating the dir here only hides the issue.